docs/workflow.rst

```diff
@@ -238,12 +233,13 @@ Let's get in more detais:
 
```

Comment:
Firstly, you must know what the meaning of “Branching Off” is . In a shorthand,  => very bad sentence, the wrong dot (.) character.